lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -388,6 +394,33 @@ def configure(conf)
       @resource.labels.freeze
       @common_labels.freeze
 
+      # If prometheus monitoring is enabled, register metrics in the default
+      # registry and store metric objects for future use. If metrics are
+      # registed already, just store the metric objects.
+      if @prometheus_monitoring_enabled
+        require 'prometheus/client'
+
+        prometheus = Prometheus::Client.registry
+
+        # Exception-driven behavior to avoid synchronization errors
+        begin
+          @prometheus_http_requests_count = prometheus.counter(
```

Comment:
Nah, even simpler -- if the prometheus library is available, use it, otherwise a no-op (which we'll later fill in with JSON).